src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -648,6 +657,8 @@ public static function cacheAndIdentifyUploadedTrack($tmp_path)
      * !This method requires the external lastfm-fpclient application to be installed on the server. A FreeBSD build
      * with URY's API key and support for -json can be found in the fpclient.git URY Git repository.
      *
+     ***** Since LastFM was removed from the central track uploader, this code MAY not be used anymore.
```

Comment:
It's used by MyRadio_FingerPrinterDaemon.php, don't want to mess anything up if that's still running somewhere...